chart/templates/benchmark-rke2-cis-1.11.yaml

```
@@ -0,0 +1,8 @@
+---
+apiVersion: compliance.cattle.io/v1
+kind: ClusterScanBenchmark
+metadata:
+  name: rke2-cis-1.11
+spec:
+  clusterProvider: rke2
+  minKubernetesVersion: "1.31.0"
```

```suggestion
  minKubernetesVersion: "1.29.0"
```
As per https://github.com/rancher/security-scan/blob/5881bf51a7497a03d94b0275af1c2cb174cdc46d/package/cfg/config.yaml#L220